checker/jdk/Makefile

```diff
@@ -18,7 +18,9 @@ all: clean classes
 # jdk.jar only contains those classes with annotations, not every class from
 # the original JDK.
 jdk.jar:
-	cd annotated; jar cf ../jdk.jar .
+	echo Implementation-Version: `git rev-parse --short HEAD` "\n" > manifest.txt
```

Comment:
I would make this "Checker-Framework-Version" or "Checker-Framework-Commit" to be more explicit.